src/checks/company/relevant-countries-all.ts

```diff
@@ -0,0 +1,20 @@
+import { CompanyCheck } from '../../types/checks';
+
+const check: CompanyCheck = {
+    id: 'relevant-countries-all',
+    desc: 'If `relevant-countries` contains `all`, it should only contain `all`.',
+    severity: 'ERROR',

```

Comment:
Providing a reference is required: 

https://github.com/datenanfragen/data/blob/ca73bc29ec6a0d1b01dd67eb58537a1600e04f03/src/types/checks.ts#L22